karabo_data/reader.py

```diff
@@ -1058,6 +1062,52 @@ def RunDirectory(path):
 RunHandler = RunDirectory
 
 
+def open_run(proposal, run, proc=True):
+    """Access data from a specified run in the EuXFEL files on Maxwell.
+
+    Parameters
+    ----------
+    proposal: str, int
+        A proposal number, such as 2012, '2012', 'p002012', or a path such as
+        '/gpfs/exfel/exp/SPB/201701/p002012'.
+    run: str, int
+        A run number such as 243, '243' or 'r0243'.
+    proc: bool
+        If True (default), combine data from the raw and 'proc' (processed)
+        folders for the specified run. If False, access only the raw data.
```

Comment:
Can we precise here that the corrected file is loaded if it exists in both raw and proc directories

Reply:
Yep, good idea. Update coming.